locale/fi/LC_MESSAGES/django.po

```diff
@@ -807,13 +841,17 @@ msgstr "Prosenttimäärä"
 msgid "Waiting"
 msgstr "Odottaa käsittelyä"
 
-msgid "Rejected"
+msgid "Paid manually"
+msgstr ""
+
+msgid "Refunded"
 msgstr "Hylätty"
 
-#, fuzzy
-#| msgid "order"
+msgid "Accepted"
+msgstr "Hyväksytty"
+
 msgid "Lease order"
-msgstr "tilaus"
+msgstr "Vuokratilaus"
```

Comment:
Not sure about this one, but I don't know what else it should be

Reply:
We don't show the word "tilaus" to customers though

Reply:
Should it be "Vuokralasku"?

Reply:
Or "Venepaikan lasku"?